syftr/__init__.py

```diff
@@ -1,3 +1,6 @@
-from importlib.metadata import version as _version
+try:
+    from importlib.metadata import version as _version
```

Comment:
Why using aliases here (`import version as _version`)

Reply:
not my idea, happy to change

Reply:
done